lib/Backends/CPU/CPUDeviceManager.h

```diff
@@ -42,8 +42,14 @@ class CPUDeviceManager : public QueueBackedDeviceManager {
 public:
   CPUDeviceManager(std::unique_ptr<DeviceConfig> config = nullptr,
                    size_t maxMemory = 2000000000)
-      : QueueBackedDeviceManager(BackendKind::CPU, std::move(config)),
-        maxMemoryBytes_(maxMemory) {}
+      : QueueBackedDeviceManager(BackendKind::CPU, std::move(config)) {
+    if (maxMemory == 2000000000 && config_ != nullptr &&
```

Comment:
If this is just a sentinel now, we should pick a more obvious value (e.g. zero).